### PR TITLE
[Hack] - Druid Http SQL integration

### PR DESCRIPTION
**Pull request by apoorvmittal10**
## Description
- PR comprises of changes for executing `druid` `sql` calls over `http post`.
- Hack to execute to `druid` queries and have utilized `thirdeye` infrastructure for `sql` query creation.
- `Avatica-Calcite` dependencies and calls have been discarded, initial approach, as we have blocked `unauthenticated` calls  of `calcite` in `druid router` and haven't enabled `broker` jdbc `authentication`.

Also, I have created another branch which shall only contain the commits for any `hack` which shall be discarded after `demo`. Just want to maintain clean `master` branch hence our initial `release/demo` shall happen from `demo_druid` branch.

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datasource/druid/DruidHttpClient.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
+package org.apache.pinot.thirdeye.datasource.druid;
```
Comment (rjain-confluent) [inline thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datasource/druid/DruidHttpClient.java:1]: can we rename this to `DruidTestHttpClient` or something similar to make it explicit that this is not to be used for production.

Comment (apoorvmittal10) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datasource/druid/DruidHttpClient.java:1]: >Also, I have created another branch which shall only contain the commits for any hack which shall be discarded after demo. Just want to maintain clean master branch hence our initial release/demo shall happen from demo_druid branch.

@rjain-confluent I am thinking to completely discard this `branch` and `PR` after demo.

Comment (apoorvmittal10) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datasource/druid/DruidHttpClient.java:1]: @rjain-confluent Shall I merge?

Comment (rjain-confluent) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datasource/druid/DruidHttpClient.java:1]: Yeah, that is fine. Please go ahead and merge.

**Pull request merged**